shared/panels/_flows/reward-flow/reward-flow.types.ts

```diff
@@ -16,5 +16,5 @@ export interface RewardFlowContextInterface {
   selectedGithubUserIds: number[];
   selectedIssueIds: string[];
   selectedContributionIds?: string[];
-  setSelectedContributionIds: (value: string[] | undefined) => void;
+  setSelectedContributionIds: ReturnType<typeof useState<string[] | undefined>>[1];
```

Comment:
why do you use returneType instead of `value: string[] | undefined) => void;` ? it's more difficult to read ?